tests/components/sighthound/test_image_processing.py

```diff
@@ -91,3 +94,19 @@ def capture_person_event(event):
     state = hass.states.get(VALID_ENTITY_ID)
     assert state.state == "2"
     assert len(person_events) == 2
+
+
+async def test_save_image(hass, mock_image, mock_detections):
+    """Save a processed image."""
+    VALID_CONFIG[ip.DOMAIN].update({sh.CONF_SAVE_FILE_FOLDER: TEST_DIR})
+    await async_setup_component(hass, ip.DOMAIN, VALID_CONFIG)
+    assert hass.states.get(VALID_ENTITY_ID)
+
+    with patch("PIL.Image.open") as pil_img:
```

Comment:
We need to patch where the target is used, not where it's defined.

`"homeassistant.components.sighthound.imageprocessing.Image.open"`

Reply:
```suggestion
    with patch("homeassistant.components.sighthound.imageprocessing.Image.open") as pil_img_open:
        pil_img = pil_img_open.return_value
```

Reply:
OK!